bin/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+app="improtresk-api"
+
+DIR_ROOT="$(dirname "$(readlink -f "$0")")"
+
+cd $DIR_ROOT
+
+source ../../bin/activate
+
+python ../manage.py migrate
```

Comment:
It is not problem when the migration fails (at least on PostgreSQL), but when it corrupts data in unexpected way, that couldn't be clearly visible at the point of running that migration.

But I assume, that your solutions are for the project of our size OK.